src/plugins/expressions/common/execution/execution.abortion.test.ts

```diff
@@ -84,7 +84,7 @@ describe('Execution abortion tests', () => {
     jest.useFakeTimers();
   });
 
-  test('nested expressions are aborted when parent aborted', async () => {
+  test.skip('nested expressions are aborted when parent aborted', async () => {
```

Comment:
👀

Reply:
Found the cause and enabled the test back.